Feature/kerberos setup edit

litzell opened the pull request:
Review at http://docs-gpdb-review-staging.cfapps.io/review/admin_guide/kerberos.html

- organize sections in a way similar to the reorg done in the HAWQ docs
- convert DITA tasks to topics to get better HTML conversion
- use consistent Kerberos realm and domain names
- use kadmin.local -q consistently instead of interactive mode sometimes
- test commands run
- add note about unsetting LD_LIBRARY_PATH to run kerberos utilities after sourcing greenplum_path.sh
- fix section on creating maps in pg_ident.conf

dyozie commented on gpdb-doc/dita/admin_guide/kerberos.xml:
```diff
@@ -21,90 +23,317 @@
+        </li>
+        <li id="nr166158">As a test, log in to the postgres database with the
+            <codeph>gpadmin/admin</codeph> role: <codeblock>psql -U "gpadmin/admin" -h mdw postgres
+psql (8.3.23)
```
Add bash shell prompt here.

litzell commented:
- added $ or # prompts for all code blocks
- Reworked section "Mapping Kerberos Principals to Greenplum Database Roles" to describe, generally, a user's authentication process and to more clearly describe how principal name is mapped to gpdb name.

mkiyama commented on gpdb-doc/dita/admin_guide/kerberos.xml:
```diff
@@ -227,41 +227,51 @@ postgres=# select current_user;
+        transformations specified in the options field of the
+          <codeph>gss</codeph> entry in the Greenplum Database
+          <codeph>pg_hba.conf</codeph> file:</p>
+      <ul id="ul_hzh_rjw_22b">
```
any reason the gss parameter krb_realm is not described here?
